spec/features/swagger_spec.rb

```diff
@@ -49,8 +49,8 @@
       it 'adds an Authorization header' do
```

Comment:
I think it'll be pretty easy. Do you want 3 tests (1 for api key, 2 for header auth) or less?